project_task_pull_request_state/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Copyright Cetmix OU 2023
```

Comment:
[__init__.py](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py) don't typically include copyright notice since there's nothing to apply copyright to.

Reply:
fixed